contracts/Vault.sol

```diff
@@ -180,19 +195,23 @@ contract Vault is ExchangeRate, Initializable, OwnableUpgradeSafe {
     /// @notice Convenience function to execute a state transition.
     function manualTransition() public {
         _stateTransition();
-        _updateCheckpointInterest();
+        if(startedSurplus) {
+            _updateCheckpointInterest();
+
+        }
     }
 
 
 
     function _stateTransition() private {
```

Comment:
general comment to us. i think this function would be very happy to be broken up into smaller functions, also easier to test each if, else clause. probably out of scope for this PR.

Reply:
agreed! `_stateTransition` function is massive. I intend to work on that next :D